### PR TITLE
non-code activity log update

### PR DESCRIPTION
**Pull request by oma131**
Signed-off-by: oma131 <anosikeihuoma@gmail.com>

### CODE REVIEW

File: NON-CODE_ACTIVITY-LOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Non-code Activity Log
+This file contains a table that shows all non-code contributions made in the CHAOSS Community.
```
Comment (Ruth-ikegah) [inline NON-CODE_ACTIVITY-LOG.md]: Hey @oma131, please can you edit this to "The CHAOSS Badging Website"? Then also add instructions like; you can make a PR to this file adding your details to the table.

**Review by Ruth-ikegah — CHANGES_REQUESTED** [review]
Hey @oma131, thanks for putting this together. I dropped a comment.

File: NON-CODE_ACTIVITY-LOG.md (hunk 1 above)
Comment (oma131) [reply NON-CODE_ACTIVITY-LOG.md]: Hello @Ruth-ikegah can you explain a bit more about the details I should add?

Comment (Ruth-ikegah) [reply NON-CODE_ACTIVITY-LOG.md]: Something along these lines: " We value and accept Non-code contributions, so we created a way for you to log in your contributions. This file contains a table that shows all non-code contributions made on the CHAOSS Badging Website.

To log in your contributions, make a Pull request to this file in the order below; thank you."

Comment (oma131) [reply NON-CODE_ACTIVITY-LOG.md]: @Ruth-ikegah I have made the changes please review them

**Review by Ruth-ikegah — APPROVED** [review]
alright this looks great now!

**Pull request merged**